packages/DropdownMenu/src/Confirmation/Confirmation.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Popover from "@paprika/popover";
+import ContentContainerStyled from "../ContentContainer.styles";
+import ConfirmationStyles from "./Confirmation.styles";
+
+const { func, string } = PropTypes;
+
+const propTypes = {
+  buttonSize: string,
+  confirmButtonType: string,
+  confirmLabel: string.isRequired,
+  description: string,
+  onCancel: func.isRequired,
+  onConfirm: func.isRequired,
+  title: string.isRequired,
+};
+
+const defaultProps = {
+  buttonSize: ShirtSizes.DEFAULT,
+  confirmButtonType: "destructive",
+  description: null,
+};
+
+const Confirmation = ({ title, buttonSize, confirmButtonType, confirmLabel, description, onConfirm, onCancel }) => {
+  const I18n = useI18n();
+  return (
+    <Popover defaultIsOpen>
```

Comment:
Needing to use another popover component here as the confirmation panel is a different width to the initial content loaded. Another option would be to re-render the outer popover in DropDownMenu.js

Reply:
Hm, I wonder if our `<Popover>` should have an imperative API to allow repositioning? I bet this is not the only use case where it would come in handy.  What do you think, @nahumzs?

Reply:
If it's sufficient to unmount the old popover and mount a new one, we can use the `key` trick to achieve that. (i.e. switching from `<Popover key="menu" />` to `<Popover key="confirmation" />` will unmount/mount as opposed to updating)

Reply:
@alexzherdev So you think it would be better to to the re-render of initial popover, Do you think there is there an advantage to either way? Performance maybe?

Reply:
We can do the key trick from now, and if we see another case for having an imperative API we can implement mike.

Reply:
I agree, for _now_ the `key` is a fine solution... but do you think I should I add and imperative API for repositioning to the list in the Popover's github issue...?

Reply:
People, I think for and against opinions and reasons would be be helpful here as anyone revisiting this PR should be able to understand why we go a certain route.

Reply:
Modifying the Popover is a bit of scope creep, changing the `key` will work and keeps you focussed on this component.  We can improve performance / improve the Popover in a later task, but it's not really the concern of this PR.

Reply:
@mikrotron Confused by this comment, My intention was never to modify the Popover, especially not for performance, I am wondering why we should favour the re-rerender of the base Popover instead of using a separate Popover for the confirmation panel. Would it be more performant to do either? My original gut feeling was to re-rerender the Popover but it would be good to discuss the reasons behind it.

Reply:
Oh sorry, I was thinking imperative API vs updating `key`.  I believe @alexzherdev's suggestion was to modify the `key` on the Popover to re-render, instead of including another Popover here.  I'll let him speak to the advantages of that approach though.

Reply:
I thought the confirmation popover was a popover inside of a popover, but seems you just want to rerender the popover but with the confirmation panel once the user clicked the delete button.

In that scenario I think would be sufficient to use the `key` replacement technique, what the key is doing is just telling react that the component is a "fresh component" so will drop any state and remove the Dom from the tree while starting a new entire react cycle is like having the popover render as it were the first time. 
[key in react](https://reactjs.org/docs/reconciliation.html#keys) and a more digest explanation) [force reset components](https://medium.com/@albertogasparin/forcing-state-reset-on-a-react-component-by-using-the-key-prop-14b36cd7448e)

What could occur is that the popover will show a small "flicker/flash" when replacing the key and loading the component again, But if this flickering is not a big of the deal, just change the key and you will be fine.

Having two popovers will let you have more control in the transition from one popover to the other one because you could control both popover's state, so you can decide whenever to close it or open it each of them.

Reply:
I've been running into problems with onBlur/focus when re-rendering one popover and rendering the confirmation panel in this way. I've created a secondary branch - https://github.com/acl-services/paprika/tree/bot-1825-dropdown-paprika-rerender - with those changes if someone want to investigate the solution further, but for iteration of the component I plan to stick with the approach of rendering the a second popover for the confirmation panel as this also is a working solution.